var/spack/repos/builtin/packages/libvterm/package.py

```diff
@@ -9,9 +9,10 @@
 class Libvterm(Package):
     """An abstract library implementation of a terminal emulator"""
     homepage = "http://www.leonerd.org.uk/code/libvterm/"
-    url      = "http://www.leonerd.org.uk/code/libvterm/libvterm-0+bzr681.tar.gz"
+    url      = "http://www.leonerd.org.uk/code/libvterm/libvterm-0.1.3.tar.gz"
 
-    version('681', sha256='abea46d1b0b831dec2af5d582319635cece63d260f8298d9ccce7c1c2e62a6e8')
+    version('0.1.3', preferred=True, sha256='e41724466a4658e0f095e8fc5aeae26026c0726dce98ee71d6920d06f7d78e2b')
+    version('0.0', sha256='6344eca01c02e2270348b79e033c1e0957028dbcd76bc784e8106bea9ec3029d', url='http://www.leonerd.org.uk/code/libvterm/libvterm-0+bzr728.tar.gz')
```

Comment:
Let's call it `0.0.0` just to be consistent with the new 3 digit numbering system.